pyproject.toml

```diff
@@ -50,7 +50,7 @@ python = ">=3.8,<3.11"
 cython = "^0.29.32"
 scipy = "^1.8.1"
 statsmodels = "^0.13.2"
-numpy = "^1.23.1"
+numpy = "==1.23.1"
```

Comment:
I don't think the `==` is strictly necessary here. With semver you can just remove the `^` symbol to match the explicit version.